frigate/config.py

```diff
@@ -1097,6 +1097,10 @@ def runtime_config(self, plus_api: PlusApi = None) -> FrigateConfig:
             elif config.objects.filters[attribute].min_score == 0.5:
                 config.objects.filters[attribute].min_score = 0.7
 
+        # auto detect hwaccel args
+        if config.ffmpeg.hwaccel_args is None:
```

Comment:
I would suggest checking for `auto` and making `auto` the default. This way users have a way to disable hwaccel if desired.